roles/neutron-data-network/tasks/ipchanged.yml

```diff
@@ -19,11 +19,18 @@
 - name: configure neutron external services failover
   template: src=etc/ipchanged/add_floating_ip mode=0755
             dest=/etc/ipchanged/{{ ansible_default_ipv4.interface }}/{{ floating_ip }}/add
+  when: neutron.acts_as_router != "true"
```

Comment:
So this would not catch somebody saying True instead of true. Probably best to check it with the bool filter.

`when: neutron.acts_as_router|bool`

Also, where is acts_as_router defined? Is it a default somewhere?